<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="filePathLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openPushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/button/music</normaloff>:/button/music</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="listenPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/button/start</normaloff>:/button/start</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="stopPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/button/Resources/stop.png</normaloff>:/button/Resources/stop.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="volumeHorizontalSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/button/Resources/pause.png</normaloff>:/button/Resources/pause.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="rewindPushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/button/Resources/rewind.png</normaloff>:/button/Resources/rewind.png</iconset>
    </property>
   </widget>
   <widget class="QSlider" name="positionHorizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>451</width>
      <height>19</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="musicList">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>141</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionjukebox"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>jukebox</string>
   </property>
  </action>
  <action name="actionjukebox">
   <property name="text">
    <string>jukebox</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
